tests/karate/src/test/java/be/rollCall/createRollCall.feature

```diff
@@ -0,0 +1,86 @@
+Feature: Create a RollCall
+  # TODO: Add missing test data
+  Background:
+    * call read('classpath:be/utils/server.feature')
+    * def createLAO = read('classpath:be/createLAO/create.feature@name=createLAO')
+    * def badMessageDataTest = read('classpath:be/common/badMessageDataRequest.feature')
+    * def valideRollCall = {}
+    * def success = {jsonrpc: "2.0", id: #(valideRollCall.id), result: 0}
+    * def socket = karate.webSocket(wsURL,handle)
+
+  @name=rollcall
+  Scenario: Create RollCall with valid request succeeds
+    Given call createLAO
+    * karate.log('RollCall: ' + karate.pretty(valideRollCall))
+    And string success = ""
+    When eval socket.send(valideRollCall)
+    And string answer = socket.listen(timeout)
+    * karate.log('Received answer = ' + answer)
+    Then match answer == success
```

Comment:
how can the answer be empty ? 🤔

Reply:
have I misunderstood something here?

Reply:
Sorry for the confusion. Test data are still missing that's all, I'll read them into the tests from `.json` files once I forge them.